atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -320,12 +320,33 @@ public Map<Cell, byte[]> get(TableReference tableRef, Set<Cell> cells) {
         }
     }
 
+    @Override
+    @Idempotent
+    public Map<Cell, byte[]> getWithExpectedNumberOfCells(
+            TableReference tableRef, Set<Cell> cells, int expectedNumberOfPresentCells) {
+        try {
+            return getWithLoader(
+                            tableRef,
+                            cells,
+                            (tableReference, toRead) -> Futures.immediateFuture(super.getWithExpectedNumberOfCells(
+                                    tableReference, toRead, expectedNumberOfPresentCells)))
+                    .get();
+        } catch (InterruptedException | ExecutionException e) {
```

Comment:
Might be missing something: usually we need to reset the interrupted flag on catching `InterruptedException` - is this different for some reason?

Reply:
Hm... not sure? I just assumed [the pattern used on the standard get method](https://github.com/palantir/atlasdb/blob/ce5995e9c0f0098e6cff531d1e687c682e7052f3/atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java#L312-L320) was correct and repeated it here?

Reply:
Oops, you're right! Fixed the `CachingTransaction` but forgot to fix the instance here.